### PR TITLE
Uar 1349

### PR DESCRIPTION
**Pull request by jshard**
Did updates (checking in original file then updates) and tested. Creating new pull request. thanks.

### CODE REVIEW

**Comment by nataliac06** [comment]
screenshot from 2016-01-25 10 22 02
Attaching the .tag diff

File: src/main/webapp/WEB-INF/tags/questionnaire/textQuestion.tag [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+<%@ attribute name="question" required="true" type="org.kuali.kra.questionnaire.question.Question" %>
+<%@ attribute name="answer" required="true" type="org.kuali.kra.questionnaire.answer.Answer" %>
+<%@ attribute name="questionIndex" required="true" %>
+
+<%@ attribute name="answerHeaderIndex" required="true" %>
+<%@ attribute name="bean" required="true" type="org.kuali.kra.questionnaire.QuestionnaireHelperBase" %>
+<%@ attribute name="property" required="true" %>
+<%@ attribute name="answerValidationError" required = "true" %>
+<c:set var="questionFieldName" value="${property}.answerHeaders[${answerHeaderIndex}].answers[${questionIndex}].answer" />
+<script language="javascript" src="scripts/ua_kuali_application.js"></script>
```
Comment (nataliac06) [inline src/main/webapp/WEB-INF/tags/questionnaire/textQuestion.tag]: You don't need to explicitly include the ua_kuali_application.js in your .tag file.
This file is set in kc-config-defaults.xml to be automatically included everywhere (in the param named kns.javascript.files)

**Comment by jshard** [comment]
Ok. removed line that included javascript file in the textQuestion.tag file.

**Comment by nataliac06** [comment]
Looks good.

**Pull request merged**